frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/selectors/template.ts

```diff
@@ -54,3 +47,14 @@ export const getTemplateValidations = (options: UpdateOptions): TemplateValidati
 
   return getValidationsFromTemplates(relevantTemplates.toArray());
 };
+
+export const getTemplateValidation = (state, id: string): TemplateValidations => {
+  const templateValidations = getTemplateValidations(state, id);
+  if (templateValidations && templateValidations.length > 0) {
+    templateValidations.length > 1 &&
+      console.warn('WARNING: getTemplateValidation: multiple template validations detected!');
```

Comment:
@suomiy @irosenzw 
If this is an error that should be handled, ensure there is something in place, otherwise, if its something that is sufficiently handled with the console statement, then please add:

```javascript 
// eslint-disable-next-line no-console
console.warn('....');
```
Otherwise the errors appear every time we build the app.
@jcaianirh

Reply:
fixed in https://github.com/openshift/console/pull/3949